### PR TITLE
Add some fixes to the campaign full

### PR DESCRIPTION
**Pull request by YosraAk**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add some fixes to the campaign full
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12067)

### CODE REVIEW

File: tests/E2E/test/selectors/BO/advancedParameters/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = Object.assign(
+  require('./performence')
```
Comment (PierreRambaud) [inline tests/E2E/test/selectors/BO/advancedParameters/index.js]: performance*

File: tests/E2E/test/selectors/BO/advancedParameters/performence.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  Performence: {
```
Comment (PierreRambaud) [inline tests/E2E/test/selectors/BO/advancedParameters/performence.js]: Performance*

Comment (YosraAk) [reply tests/E2E/test/selectors/BO/advancedParameters/performence.js]: Thanks! it was a typo error :sweat_smile:

File: tests/E2E/test/campaigns/common_scenarios/product.js [hunk 3]
```diff
@@ -55,8 +55,9 @@ module.exports = {
       test('should click on "New Product" button', () => {
         return promise
           .then(() => client.waitForExistAndClick(AddProductPage.new_product_button))
+          .then(() => client.isVisible(AddProductPage.symfony_toolbar, 3000))
           .then(() => {
-            if (global.ps_mode_dev) {
+            if (global.ps_mode_dev && global.isVisible) {
```
Comment (PierreRambaud) [inline tests/E2E/test/campaigns/common_scenarios/product.js]: You maybe need to create an helper to remove this duplicate call. Like `client.waitForSymfonyToolbar(AddProductPage)`
and this function can contain the check about ps_mode_dev and isVisible and the client.waitForExsistAndClick :)

**Review by mbadrani — APPROVED** [review]
PR so anticipated

**Pull request merged**